plugins/inputs/logparser/grok/grok.go

```diff
@@ -186,6 +186,7 @@ func (p *Parser) Compile() error {
 
 // ParseLine is the primary function to process individual lines, returning the metrics
 func (p *Parser) ParseLine(line string) (telegraf.Metric, error) {
+
```

Comment:
Undo these whitespace changes, so that git history will be a little easier to read.